tests/sentry/integrations/vsts/test_client.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import absolute_import
+
+import responses
+
+from time import time
+
+from sentry.models import Identity, IdentityProvider, Integration
+from .testutils import VstsIntegrationTestCase
+
+
+class VstsApiClientTest(VstsIntegrationTestCase):
```

Comment:
:o thanks for the test!

Reply:
I just moved a test you wrote for the client into it's own spec 😛